.vscode/tasks.json

```diff
@@ -66,8 +66,25 @@
 				}
 			],
 			"dependsOn": [
-				"build debug using make"
+				"clean",
```

Comment:
It will cause the whole project compile again. In big projects it will significantly increase the compilation times.